payment_processor.js

```
@@ -0,0 +1,359 @@
+    saveReceipt(transactionId, data, filename) {
+        // No path sanitization
+        const path = `./receipts/${filename}`;
+        fs.writeFileSync(path, data);
+        
+        return { saved: path };
+    }
+
+    // Unsafe deserialization
+    importTransactions(data) {
+        // Using eval to parse data - extremely dangerous
+        const transactions = eval('(' + data + ')');
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)